jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/BadgeTests.java

```diff
@@ -1,85 +1,64 @@
 package io.github.epam.material.tests.displaydata;
 
+import io.github.epam.TestsInit;
+import io.github.epam.test.data.BadgeDataProvider;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
 import static io.github.com.StaticSite.badgePage;
 import static io.github.com.pages.displaydata.BadgePage.badgeDifferentPosition;
 import static io.github.com.pages.displaydata.BadgePage.buttonIncrease;
 import static io.github.com.pages.displaydata.BadgePage.buttonReduce;
-import static io.github.com.pages.displaydata.BadgePage.dotBadge;
-import static io.github.com.pages.displaydata.BadgePage.errorColorBadge;
-import static io.github.com.pages.displaydata.BadgePage.primaryColorBadge;
-import static io.github.com.pages.displaydata.BadgePage.secondaryColorBadge;
+import static io.github.com.pages.displaydata.BadgePage.dotBadgeContainer;
+import static io.github.com.pages.displaydata.BadgePage.secondaryColorBadgeContainer;
 import static io.github.com.pages.displaydata.BadgePage.switchShowBadge;
 import static io.github.com.pages.displaydata.BadgePage.switchShowZero;
 
-import com.epam.jdi.light.material.elements.displaydata.Badge;
-import io.github.epam.TestsInit;
-import org.testng.annotations.BeforeMethod;
-import org.testng.annotations.DataProvider;
-import org.testng.annotations.Test;
-
 public class BadgeTests extends TestsInit {
 
-    @DataProvider
-    public static Object[][] alignmentBadges() {
-        return new Object[][]{
-                {badgeDifferentPosition.get(1), "-1", "TopRightRectangle"},
-                {badgeDifferentPosition.get(2), "5", "BottomRightRectangle"},
-                {badgeDifferentPosition.get(3), "6", "BottomLeftRectangle"},
-                {badgeDifferentPosition.get(4), "7", "TopLeftRectangle"},
-        };
-    }
-
     @BeforeMethod
     public void before() {
         badgePage.open();
         badgePage.isOpened();
     }
 
-    @Test
-    public void simpleBadgeTest() {
-        primaryColorBadge.is().displayed()
-                .and().has().text("4")
-                .and().position("TopRightRectangle");
-
-        errorColorBadge.is().displayed()
-                .and().has().text("4")
-                .and().position("TopRightRectangle");
-    }
-
     @Test
     public void variousBadgeTest() {
-        secondaryColorBadge.is().displayed();
-        secondaryColorBadge.has().text("1")
+        secondaryColorBadgeContainer.icon().is().displayed();
+        secondaryColorBadgeContainer.badge().is().displayed()
+                .and().has().text("1")
```

Comment:
One test might have a color check. But don't use PRIMARY and SECONDARY enum items